examples/example-gateway/idl/clients/baz/baz.thrift

```diff
@@ -39,7 +39,7 @@ service SimpleService {
 }
 
 service SecondService {
-    void Echo(
+    string Echo(
```

Comment:
Can we add a new method to http bar client that returns a string to verify this.